### DISCUSSION TITLE
Documentation Page

### DISCUSSION THREAD

**Discussion by jankapunkt** [meteor/blaze#430] [Ideas]
The documentation on https://blazejs.org is using hexo but we might move to something else for various reasons:

- searchability
- theming (dark theme detection)
- better integrations with CI/CD
- better inteagrations with custom themeing

Proposals? Ideas?

**Comment by StorytellerCZ** [comment]
The current de-facto standard for documentation sites is [Docusaurus](https://docusaurus.io/).

In this I would like to also add I18n.

**Comment by jankapunkt** [comment]
I would rathjer vote for vitepress because its using Vue, which is very close to Blaze and very easy to setup/maintain. @italojs if you like the idea, I would start a PR to rewrite the documentation using vitepress and add a ci workflow to auto-deploy the docs.

**Reply by nachocodoner** [reply]
Our new Meteor 3 and migration docs are also written with VitePress in the Meteor core repository. Since we support Vue in Meteor, it should be a valid option, just like a docs platform closed to React.

I’m not too familiar with Vue, but other maintainers can handle it.

I’m fine moving in that direction, but I’d like a second opinion from @italojs.

**Reply by italojs** [reply]
I'm sorry for the delay, i'm cleaning up my notifications and just saw it now
IMO Vue is good, it's simple to maintain

**Reply by jankapunkt** [reply]
Related PR is #494
